### PR TITLE
GBFS Governance Revisions

### LINKED ISSUE #556: GBFS Governance Working Document and Release Cycles
In 2020 MobilityData revised the GBFS Governance Process, since then we have noticed some elements of confusion and ambiguity. At the 2023 North American Workshop in New York City, we worked on a revision document to begin keeping track of all of things that need more attention. 

MobilityData would like to open this discussion to the broader community for feedback. You can find the working document [here](https://docs.google.com/document/d/1kRJnJyHT7N7gUxHktzvLE3x9kotA7hI_bkP10Lh8d_c/edit?usp=sharing) that is open for comment. This document will remain the source of truth for changes until a PR is opened, and this issue is closed.  

While the entire governance process is up for review, there are some key questions we would like some feedback on: 
1. What should the conditions be for blocking a proposal? Currently, one person is able to block a proposal, should the threshold be higher? 
2. The frequency of releases - We are seeing the official release v3.0 being blocked until a release and adoption v3.0-RC2 (and other changes in v3.0-RC), the release of v3.0-RC2 is blocked because more and more breaking proposals are being opened - which is great! That said, we would like to create a release timeline and release cycle that is predictable. We would like to propose distinct periods in which we accept proposals for a version, with a cut off date. The intention is for producers and consumers to be well aware ahead of time when a new version will be released, to help them better plan their work. Currently, MAJOR versions are released no more than once every 12 months. We propose to choose a specific date, for example the 1st of a chosen month, where at that point no more changes will be accepted, and the voting period begins for all remaining open PRs (and any required discussion periods). At that point, everyone can expect a MAJOR version to be released by the end of that month, every year. We would do the same with MINOR versions, say every 6 months, which would mean 2 releases a year, one MINOR and one MAJOR.   

Please let us know what you think about these changes in this thread or in the document linked above! ([Here](https://docs.google.com/document/d/1kRJnJyHT7N7gUxHktzvLE3x9kotA7hI_bkP10Lh8d_c/edit?usp=sharing) again for your convenience).

Thank you in advance!

### PR DESCRIPTION
**Pull request by richfab**
Fixes https://github.com/MobilityData/gbfs/issues/556

This PR contains the following changes to the governance:
- Changes to the governance are now subject to a vote.
- Versions are now released on pre-determine dates:
    - MINOR version release every May
    - MAJOR version release every November.
- Pull Requests and Issues now become stale after 60 days (was 120 days) and are closed 30 days later (was 60 days).
- Define the role of the Maintainer.
- Make explicit that votes from multiple members of the same organization are counted as one vote.
- Make explicit that editorial changes during the voting process are allowed.
- Use [RFC2119](https://tools.ietf.org/html/rfc2119), [BCP 14](https://tools.ietf.org/html/bcp14) and [RFC8174](https://tools.ietf.org/html/rfc8174) standard for key words such as "MUST", "MAY", etc.

For your convenience, you may visualize the changes to the governance in the working document [here](https://docs.google.com/document/d/1kRJnJyHT7N7gUxHktzvLE3x9kotA7hI_bkP10Lh8d_c/edit#heading=h.wvusrhrxh07r).

### CODE REVIEW

**Review by josee-sabourin — CHANGES_REQUESTED** [review]
LGTM - we just need to update the README to make sure things are not in two places (I'm thinking specifically of the section of Versioning).

**Comment by mplsmitch** [comment]
The MINOR release line is puzzling to me. 

> MINOR releases SHOULD be released every May and November by the Maintainer.

The only time you would have a MINOR release in November is if there is no MAJOR release for that year. For example you wouldn't have a v4.0 MAJOR and a v4.1 MINOR in November.

A simpler way to say this would be:

> 
> - MINOR versions SHOULD be released every May
> - MAJOR versions SHOULD be released every November
> - In the event that there is no MAJOR version release for the period, a MINOR version MAY be released in November.

**Review by josee-sabourin — APPROVED** [review]

**Comment by richfab** [comment]
I hereby call a vote on this proposal. Voting will be open for 10 full calendar days until 11:59PM UTC on Saturday, January 13, 2024.
Please vote for or against the proposal, and include the organization for which you are voting in your comment.

File: governance.md [hunk 1]
```diff
@@ -1,31 +1,61 @@
 ## Governance Process
 
-GBFS is an open specification, developed and maintained by the community of producers and consumers of GBFS data.
-The specification is not fixed or unchangeable. As the shared mobility industry evolves, it is expected that the specification will be extended by the GBFS community to include new features and capabilities over time. If you are new to engaging with the community on this repository, firstly welcome! Please identify which organization you represent when posting.
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](https://tools.ietf.org/html/rfc2119), [BCP 14](https://tools.ietf.org/html/bcp14) and [RFC8174](https://tools.ietf.org/html/rfc8174) when, and only when, they appear in all capitals, as shown here.
```
Comment (testower) [inline governance.md:3]: "key words" => "keywords"

Comment (mplsmitch) [reply governance.md:3]: I agree, however, that text  including "key words" comes directly from rfc2119 and rfc1874

Comment (testower) [reply governance.md:3]: I see dictionaries don't agree among themselves either. I'm fine with just leaving it as is

**Comment by futuretap** [comment]
+1 from Where To?

**Comment by cmonagle** [comment]
+1 from Transit

**Comment by richfab** [comment]
Voting on this PR closes in 2 calendar days. Please vote for or against the proposal, and include the organization for which you are voting in your comment.

**Comment by BredeD** [comment]
+1 from Entur

**Comment by richfab** [comment]
This vote has now closed, and it passes!

Votes in favor:
Where To? (consumer)
Transit app (consumer)
Entur (producer)
 
There were no votes against.

This change will take effect immediately. Note that this does not affect v3.0-RC2 which is already released and ready for implementation!

**Review by davidgamez — APPROVED** [review]

**Pull request merged**